packages/walletconnect/src/index.spec.ts

```diff
@@ -35,7 +35,7 @@ describe('WalletConnect', () => {
     beforeEach(async () => {
       let actions: Actions
       ;[store, actions] = createWeb3ReactStoreAndActions()
-      connector = new WalletConnect({ actions, options: { rpc: {} } })
+      connector = new WalletConnect({ actions, options: { projectId: '', chains: [1] } })
```

Comment:
maybe make the chainID a const above?

Reply:
yeah, I have to write tests for this file, will update!